repo/packages/W/weavescope/0/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "2.0",
+  "name": "weavescope",
+  "version": "0.13.1",
+  "tags": ["mesosphere", "monitoring", "marathon", "docker", "microservices", "control", "visualisation", "development"],
```

Comment:
these tags are used to distinguish your service.  you should not use `mesosphere`, `mesos`, `marathon` or `docker` here.  I'm not sure development makes sense either.   control seems strange as well.   visualization, debugging, monitoring make sense.